servers/partner-ci/10.0/zabbix.yaml

```diff
@@ -0,0 +1,366 @@
+    disabled: false
+    node: zabbix
+    <<: *test-parameters
+    scm:
+      - common-scm:
+          scm_submodule_disabled: '{scm_submodule_disabled}'
+          scm_wipe_workspace: '{scm_wipe_workspace}'
+          gerrit_repo: '{gerrit_repo}'
+          skip_vote_successfull: '{skip_vote_successfull}'
+          skip_vote_failed: '{skip_vote_failed}'
+    wrappers:
+      - test-wrapper
+    properties:
+      - test-property
+    builders:
+      - 10-get-refs
+      - get-props
+      - get-ubuntu-mirrors
+      - conditional-step:
+           condition-kind: shell
+           condition-command: |
+            [ "${{BUILD_PKG}}" == "true"       ]
+           steps:
+             - trigger-builds:
+                - project: '{fuel_release}.{zbx_extreme_net_repo}.build'
+                  current-parameters: false
+                  block: true
+                - project: '{fuel_release}.{zbx_emc_repo}.build'
+                  current-parameters: false
+                  block: true
+                - project: '{fuel_release}.{zbx_snmp_repo}.build'
+                  current-parameters: false
+                  block: true
+                - project: '{fuel_release}.{zbx_ext_repo}.build'
+                  current-parameters: false
+                  block: true
+      - copyartifact:
+          - project: '{fuel_release}.{zbx_extreme_net_repo}.build'
+            which-build: last-successful
+          - project: '{fuel_release}.{zbx_emc_repo}.build'
+            which-build: last-successful
+          - project: '{fuel_release}.{zbx_snmp_repo}.build'
+            which-build: last-successful
+          - project: '{fuel_release}.{zbx_ext_repo}.build'
+            which-build: last-successful
+      - shell:
+          !include-raw-escape: ../builders/common.prepare.sh
+      - shell:
+          !include-raw-escape: ../builders/zabbix.test.sh
+    publishers:
+      - runner-publisher:
+          properties_name: properties
+          email_to: '{email_to}'
+      - test-publisher:
+          reporter_name: '{reporter-name}'
+
+- job-template:
+    name: '{fuel_release}.{plugin_name}.{testgroup}'
+    description: 'The test "{testgroup}" for {fuel_release}.{plugin_name}'
+    concurrent: true
+
+- job-template:
+    name: '{fuel_release}.{plugin_name}.smoke.test'
+    description: 'The Smoke test for {fuel_release}.{plugin_name}'
+    concurrent: true
+
+- job-template:
+    name: '{custom-name}.qa'
+    description: 'The custom test for {fuel_release}.{plugin_name}'
+    concurrent: true
+
+- job-template:
+    name: '{custom-name}.dev'
+    description: 'The custom test for {fuel_release}.{plugin_name}'
+    concurrent: true
+
+- job-template:
+    name: '{nightly-name}'
+    project-type: multijob
+    disabled: '{obj:nightly_disabled}'
+    node: runner
+    <<: *runner-parameters
+    scm: []
+    triggers:
+      - timed: '{nightly_timer}'
+    properties:
+      - runner-property:
+          build-name: '{build-name}'
```

Comment:
seems we need to replace it with ui-runner-property
and definitely we need to rename ui-runner-property